scripts/config_check.py

```diff
@@ -30,13 +33,55 @@ def look_for_password(dictionary):
                                 '"REPLACE_WITH_REAL_USERNAME"')
 
         if isinstance(val, dict):
-            look_for_password(val)
+            look_for_password_in_json(val)
+
+
+def look_for_passwords_in_shell_script(lines):
+    """
+    Check for dangerous plaintext passwords in a text file organized by lines.
+    Essentially performs a grep for USERNAME= and PASSWORD= and makes sure they
+    have well-known standard values REPLACE_WITH_REAL_USERNAME, etc.
+
+    :param list lines: A list of lines in the script file.
+    """
+
+    un_template = 'REPLACE_WITH_REAL_USERNAME'
+    pw_template = 'REPLACE_WITH_REAL_TEMPLATE'
```

Comment:
Non-blocking nitpick:  out of curiosity, why "REPLACE_WITH_REAL_TEMPLATE" instead of "REPLACE_WITH_REAL_PASSWORD"?

Also, if it's "REPACE_WITH_REAL_PASSWORD", the same template variables could be used for both this function and `look_for_pasword_in_json`.